pumps/sql_aggregate_test.go

```diff
@@ -326,3 +335,105 @@ func TestDecodeRequestAndDecodeResponseSQLAggregate(t *testing.T) {
 	assert.False(t, newPump.GetDecodedRequest())
 	assert.False(t, newPump.GetDecodedResponse())
 }
+
+func TestEnsureIndex(t *testing.T) {
```

Comment:
is it possible to add any failure scenario? If not, remove the `expectedErr` field

Reply:
added